src/Interfacer.jl

```diff
@@ -175,6 +175,11 @@ abstract type SeaIceModelSimulation <: SurfaceModelSimulation end
 abstract type LandModelSimulation <: SurfaceModelSimulation end
 abstract type OceanModelSimulation <: SurfaceModelSimulation end
 
+# Simulation objects tend to be very big, so it is best to make sure they are not printed in the REPL
+function Base.show(io::IO, sim::ComponentModelSimulation)
```

Comment:
```suggestion
function Base.show(io::IO, Base.@nospecialize(sim::ComponentModelSimulation))
```
since the implementation doesn't depend on anything in `sim`.